docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -1527,6 +1527,25 @@ Exec=sh -c "sleep inf"
 Pod=test.pod
 ```
 
+Example `s3fs.volume`:
+
+Check out [s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse)
+
+> NOTE: `use_cache` is UNBOUNDED, be careful.
+
+```
+[Service]
+CacheDirectory=s3fs
+ExecStartPre=/usr/local/bin/aws s3api put-object --bucket ${S3_BUCKET} --key ${PATH}/
```

Comment:
Please indicate where `$S3_BUCKET` and `$AWS_REGION` and `${CACHE_DIRECTORY}` (below) are coming from, and/or if you need magic `Env` directives, or if they're meant to be replaced by the user, or something else

Reply:
There are a lot of different ways to set a systemd environment variable, I thought it prudent to keep the example as clean as possible.

I wanted to make it obvious which values you need to replace.

---

`CACHE_DIRECTORY` is a special variable that gets set when you specify [`CacheDirectory=`](https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#RuntimeDirectory=), but can also be over-ridden by the service environment

Depending on your environment `AWS_REGION` might already be set globally.

For the other ones, there are a lot of different ways to enrich the systemd environment 

You could do it with in the service itself
```ini
[Service]
...
Environment=S3_BUCKET=blalblalbla
Environment=PATH=something/somewhere
Environment=AWS_REGION=aq-central-1
...
```

You could do it with a Environment File
```ini
EnvironmentFile=/somewhere/s3fs-settings.env
```

You could do it with a systemd drop-in
/run/systemd/system/s3fs.service.d/10-defaults.conf
```ini
[Service]
Environment=S3_BUCKET=blalblalbla
Environment=PATH=something/somewhere
Environment=AWS_REGION=aq-central-1
```

Maybe you need to setup multiple and decide to use template directives instead:
```ini
Environment=S3_BUCKET=%i
```

---

If you'd like, I can replace the env variables with hard values instead, like:
```ini
[Service]
CacheDirectory=s3fs
ExecStartPre=/usr/local/bin/aws s3api put-object --bucket my-s3-bucket --key my-subpath/

[Volume]
Device=${S3_BUCKET}:/${PATH}
Type=fuse.s3fs
VolumeName=s3fs-volume
Options=iam_role,endpoint=aq-central-1,use_xattr,listobjectsv2,del_cache,use_cache=${CACHE_DIRECTORY}
# `iam_role` assumes inside EC2, if you are not, Use `profile=` instead
```

Or should I just specify which variables the user should set instead?

Reply:
I added a little paragraph on the variables that should be set by the user